schemas/rum/_common-schema.json

```diff
@@ -168,6 +168,30 @@
         }
       }
     },
+    "display": {
+      "type": "object",
+      "description": "Display properties",
+      "properties": {
+        "viewport": {
```

Comment:
It will be used to filter clickmaps by viewport size. @xgouchet mention that it could also be interesting for mobile. Do you have something close to this notion?

Reply:
after the discussion with @amortemousque it is clear that `viewport` is just a rectangular area defining the user visible content (some content may be outside this area in case of the scrollable content). While `viewport` is not the term commonly used on mobile, I think we can still use it (and keep the name as is).

Reply:
I've updated the definition. Let me know what you think.